.github/workflows/test.yml

```diff
@@ -147,7 +147,7 @@ jobs:
             name: PyQt6
 
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v3
```

Comment:
Why this downgrade?

Reply:
Thanks for checking. It probably remained the same from the previous version of PR. Fixed now.